library/linux/src/embedder.cc

```diff
@@ -50,6 +51,7 @@ struct FlutterEmbedderState {
   // deleted from the heap.
   std::vector<flutter_desktop_embedding::KeyboardHookHandler *>
       keyboard_hook_handlers;
+  std::unique_ptr<flutter_desktop_embedding::KeyEventHandler> key_event_handler;
```

Comment:
Please add a quick comment explaining what this is, as you would with a class ivar.